### ISSUE TITLE
Eventually move ciso_on into marbl_interface_class

### ISSUE THREAD

**Issue by mnlevy1981** [marbl-ecosys/MARBL#10]
We discussed today the idea of a `marbl_init_type` datatype, that would be set in `marbl_instance%init()` and referenced in `marbl_instance%set_interior()` and `marbl_instance%set_surface_forcing()`. It would definitely contain `ciso_on`, which is currently part of the driver because much of the `if (ciso_on) then` logic is still in the driver rather than pushed into the subroutines in `marbl_interface_class`. So ideally `ciso_on` will be part of the `marbl_interface_class` (which will involve moving it from `ecosys_driver_nml` to `ecosys_nml`).

Before we can do that we need to clean up the driver... which will be difficult due to the indexing scheme used by the tracers. Sometimes the CISO tracers are `1:ecosys_ciso_tracer_cnt`; other times they start the indexing at `ecosys_tracer_cnt + 1`. Perhaps that move from the driver to MARBL should be a separate issue?

Labelling this a bug, but priority is somewhat low given the dependence on cleaning up the driver code.
